serverlessworkflow/modules/ROOT/pages/getting-started/modules/create-your-first-workflow-service.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+:description: Creating first workflow service
+:keywords: creating, serverless, workflow
+:navtitle: Creating your first workflow service
+:page-aliases:
+
+[id="create-your-first-workflow-service_{context}"]
+= Creating your first workflow service
+
+As a developer, you can use {context} and create a `Hello World` application, which includes the following procedures:
+
+* xref:getting-started/modules/proc-boostrapping-the-project.adoc[Bootstrapping a project]
+* xref:getting-started/modules/proc-creating-workflow.adoc[Creating a workflow]
+* xref:getting-started/modules/proc-running-application.adoc[Running your workflow application]
+* xref:getting-started/modules/proc-testing-application.adoc[Testing your workflow application]
```

Comment:
Those should be "internal" links corresponding to titles, like for example, for the first link: 
```
#proc-boostrapping-the-project_Serverless%20Workflow
```
else you get redirected to the module page and have no access to the rest of the main page.